### PR TITLE
LGElectronics.SmartShare version 33.2.3.1511.1201

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98699)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @upintheairsheep,

The installation isn't unattended:
image

Is there an installer switch to bypass this and have it install automatically?

**Comment by upintheairsheep** [comment]
> Hi @upintheairsheep,
> 
> The installation isn't unattended: image
> 
> Is there an installer switch to bypass this and have it install automatically?

Try https://docs.revenera.com/installshield26helplib/helplibrary/InstallShieldSilent.htm

File: manifests/l/LGElectronics/SmartShare/33.2.3.1511.1201/LGElectronics.SmartShare.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: LGElectronics.SmartShare
+PackageVersion: 33.2.3.1511.1201
+PackageLocale: en-US
+Publisher: LG Electronics
+PackageName: LG SmartShare
```
Comment (stephengillie) [inline manifests/l/LGElectronics/SmartShare/33.2.3.1511.1201/LGElectronics.SmartShare.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.lg.com/us
PublisherSupportUrl: https://www.lg.com/us/support
PrivacyUrl:   https://privacy.us.lg.com/
PackageName: LG SmartShare
PackageUrl: https://www.lg.com/us/support/help-library/smart-share-downloads-CT10000018-1427120133847
```

File: manifests/l/LGElectronics/SmartShare/33.2.3.1511.1201/LGElectronics.SmartShare.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: LGElectronics.SmartShare
+PackageVersion: 33.2.3.1511.1201
+PackageLocale: en-US
+Publisher: LG Electronics
+PackageName: LG SmartShare
+License: Proprietary
+ShortDescription: smart tv lg
```
Comment (stephengillie) [inline manifests/l/LGElectronics/SmartShare/33.2.3.1511.1201/LGElectronics.SmartShare.locale.en-US.yaml:10]:
```suggestion
ShortDescription: LG TVs - Smart Share
```

File: manifests/l/LGElectronics/SmartShare/33.2.3.1511.1201/LGElectronics.SmartShare.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: LGElectronics.SmartShare
+PackageVersion: 33.2.3.1511.1201
+Installers:
+- InstallerUrl: http://ngfts.lge.com/fts/gftsDownload.lge?biz_code=LGDLNA&func_code=LGDLNA&file_path=/lgdlna/lgdlna/LG_SmartShare_WAL_33_2.3.1511.1201.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
+  NestedInstallerFiles:
+  - RelativeFilePath: setup.exe
```
Comment (stephengillie) [inline manifests/l/LGElectronics/SmartShare/33.2.3.1511.1201/LGElectronics.SmartShare.installer.yaml:12]:
```suggestion
  - RelativeFilePath: setup.exe /s
```

File: manifests/l/LGElectronics/SmartShare/33.2.3.1511.1201/LGElectronics.SmartShare.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  InstallerSha256: BF286A959CB6AB8C1527D955C2A86282109F9B10B09B082ABE89762313551F10
```
Comment (stephengillie) [inline manifests/l/LGElectronics/SmartShare/33.2.3.1511.1201/LGElectronics.SmartShare.installer.yaml:13]:
```suggestion
  InstallerSha256: BF286A959CB6AB8C1527D955C2A86282109F9B10B09B082ABE89762313551F10
  InstallerSwitches:
    Silent: -s
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**